src/Enum.php

```diff
@@ -147,6 +152,7 @@ protected static function resolveValuesFromStaticMethods(ReflectionClass $static
             $selfStaticMethods[$method->name] = $method->name;
         }
 
+        $enumValues = [];
```

Comment:
Why move this variable?

Reply:
Looks better for me to place declarations closer to usages. Reverted.